cookie.ts

```diff
@@ -114,3 +112,8 @@ function encodeCookieOptions (options : {[key: string]: string|boolean|null}) :
 
     return encodedOptions.join(" ;");
 }
+
+function escapeStringRegexp (text: string) : string
```

Comment:
This is a general purpose function and should be placed in a separate file, with a doc comment, tests and be exported.